test/gpu/native/basics/reductionNoGpuize.good

```diff
@@ -0,0 +1,2 @@
+warning: The prototype GPU support implies --no-checks. This may impact debuggability. To suppress this warning, compile with --no-checks explicitly
```

Comment:
You should make sure that you're running this on `main`, and that you have `CHPL_TEST_GPU` set when running start_test. This warning should no longer appear in good files.